test/rendering/keep_lines_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule RenderKeepLinesTest do
+  use ExUnit.Case
+
+  import Slime, only: [render: 1]
+
+  setup do
```

Comment:
Could you please explain, how to use it in setup block? I don't want to repeat env setting in each test.